swarm/metrics/flags.go

```diff
@@ -89,11 +92,15 @@ func Setup(ctx *cli.Context) {
 			password               = ctx.GlobalString(MetricsInfluxDBPasswordFlag.Name)
 			enableExport           = ctx.GlobalBool(MetricsEnableInfluxDBExportFlag.Name)
 			enableAccountingExport = ctx.GlobalBool(MetricsEnableInfluxDBAccountingExportFlag.Name)
+			datadir                = ctx.GlobalString("datadir")
 		)
 
 		// Start system runtime metrics collection
 		go gethmetrics.CollectProcessMetrics(4 * time.Second)
 
+		// Start collecting disk metrics
+		go datadirDiskUsage(datadir, 4*time.Second)
```

Comment:
i think that every 10 seconds is also fine

Reply:
The reason I use 4*time.Second is that the InfluxDB reporter resolution is 10*time.Second and I want to be 100% sure we have at least 1 measurement in every resolution. Otherwise the graphs could have gaps in Grafana.